app/views/step_by_step_pages/index.html.erb

```diff
@@ -35,12 +35,13 @@
         </th>
         <td>
           <ul class="list-inline">
-            <li><%= link_to 'Edit content', step_by_step_page %></li>
+            <li><%= link_to 'Edit', step_by_step_page %></li>
+            <li><%= link_to 'Choose navigation', step_by_step_page_navigation_rules_path(step_by_step_page) %></li>
             <% if step_by_step_page.has_draft? %>
-              <li><%= link_to 'Preview', preview_url(step_by_step_page.slug) %></li>
+                <li><%= link_to 'Preview', preview_url(step_by_step_page.slug) %></li>
             <% end %>
             <% unless step_by_step_page.has_been_published? %>
-              <li><%= link_to 'Delete', step_by_step_page, method: :delete, data: { confirm: 'Delete this step by step page?' } %></li>
+                <li><%= link_to 'Delete', step_by_step_page, method: :delete, data: { confirm: 'Are you sure?' } %></li>
```

Comment:
I wonder whether we should keep `Delete this step by step page?` as we want people to be doubly sure when they're doing a destructive thing. (I know this was a merge thing and not actually you :-) )

Reply:
I don't know the answer on this one